src/main/java/org/solid/testharness/http/HttpConstants.java

```diff
@@ -0,0 +1,41 @@
+package org.solid.testharness.http;
```

Comment:
I went with HttpConstants because it is the constants in the http package which is where I handle the HttpClient interactions. I've seen others do similar e.g. https://github.com/trellis-ldp/trellis/blob/main/core/common/src/main/java/org/trellisldp/common/HttpConstants.java so unless you think it is very misleading I am inclined to leave it as it is.